modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/StoriesFragment.kt

```diff
@@ -0,0 +1,44 @@
+package au.com.shiftyjelly.pocketcasts.endofyear
+
+import android.graphics.Color
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.platform.ViewCompositionStrategy
+import au.com.shiftyjelly.pocketcasts.compose.AppTheme
+import au.com.shiftyjelly.pocketcasts.ui.R
+import au.com.shiftyjelly.pocketcasts.ui.helper.StatusBarColor
+import au.com.shiftyjelly.pocketcasts.views.fragments.BaseDialogFragment
+
+class StoriesFragment : BaseDialogFragment() {
+    override val statusBarColor: StatusBarColor
+        get() = StatusBarColor.Custom(Color.BLACK, true)
+
+    override fun onCreate(savedInstance: Bundle?) {
+        super.onCreate(savedInstance)
+        setStyle(STYLE_NORMAL, R.style.BottomSheetDialogThemeBlack)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View {
+        return ComposeView(requireContext()).apply {
+            setContent {
+                AppTheme(theme.activeTheme) {
```

Comment:
Just wanted to confirm that you meant to use `AppTheme` here. I noticed that in `StoriesScreenPreview` you're using `AppThemeWithBackground`.

Reply:
I used `AppTheme` instead of `AppThemeWithBackground` here as we don't want to use surface but black background on Stories screen.

I replaced `StoriesScreenPreview` theme to match this in 38559e4.